src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -216,7 +218,7 @@
     [:div {}
      [comps/Blocker {:banner (:blocker @state)}]
      [:div {:style {:padding "1em 2em"}}
-      (render-side-bar state refs config editing? props)
+      (render-side-bar this state refs config editing? props)
```

Comment:
If the only thing you're using from `this` is the one method, why not just pass that method directly here?

Also, not your fault, but this is an awfully large number of positional arguments. You might pass them with `utils/restructure` here and pull them out with `:keys` on the other end, but it's not a huge deal, as these are all private methods...